controllers/ItemsController.php

```diff
@@ -424,6 +424,15 @@ public function items() {
 						$this->permissions
 					);
 				}
+				// Unfiled items
+				else if ($this->subset == 'unfiled') {
+					$this->allowMethods(array('GET'));
+					
+					$title = "Unfiled items";
+					$itemIDs = Zotero_Items::getItemsWithoutCollection(
+						$this->objectLibraryID
```

Comment:
This will need to use `Zotero_Items::search()` like `trash` above, since otherwise other query parameters don't work. (E.g., people still need to be able to use the search bar, which uses `q`.) `Zotero_Items::search()` is quite convoluted, for complicated reasons, so let me know if something is unclear.

`/unfiled` also implies `/top`, but I think just setting `$onlyTopLevel = true` (the second parameter to `search()`) won't work, since that matches parent items of matching items, and all child items are technically unfiled, so it would match all items with children. So instead of that, the actual unfiled JOIN condition in `search()` should probably include a negative match on `itemTopLevel.itemID`, which contains child items mapped to top-level items, so that the actual matches are only top-level items. You can test, but I think this would be the unincorporated version of the SQL:

```sql
SELECT * FROM items I
    LEFT JOIN itemTopLevel ITL USING (itemID)
    LEFT JOIN collectionItems CI USING (itemID)
    WHERE libraryID=?
    AND ITL.itemID IS NULL
    AND CI.collectionID IS NULL
```